docs/UserGuide.md

```diff
@@ -79,6 +79,32 @@ Example:
 * `add i/A0123456Z n/john m/CS2103T p/98765432 t/@john e/E0123456Z`
     * A student named John is added to TAPA.
 
+### Deleting a student: `delete`
+
+Deletes a student from TAPA.
+
+Format: `delete STUDENT_INDEX`
+
+* The student whose index in the address book is specified after the `delete` command will be removed from the address book.
```

Comment:
```suggestion
* The student corresponding to the index (specified after the `delete` command) will be removed from TAPA.
```
Just a suggestion since it feels a little weird to repeat address book (or TAPA) twice in a single sentence.